src/main/java/org/alfresco/repo/security/authentication/AuthenticationServiceImpl.java

```diff
@@ -42,7 +43,10 @@
     private Log logger = LogFactory.getLog(AuthenticationServiceImpl.class);
     AuthenticationComponent authenticationComponent;
     TicketComponent ticketComponent;
-    
+
+    /** a uuid identifying this unique instance */
+    private String uuid;
```

Comment:
The variable name does not really describe what it does.
Please use something like: `serviceInstanceId`

Reply:
I will update it, thanks.